src/server/bg_workers.js

```diff
@@ -72,18 +71,9 @@ function register_rpc() {
     });
 }
 
-
-function register_bg_worker(worker, run_batch_function) {
-    dbg.log0('Registering', worker.name, 'bg worker');
-    if (run_batch_function) {
-        worker.run_batch = run_batch_function;
-    }
-    if (!worker.name || !_.isFunction(worker.run_batch)) {
-        console.error('Name and run function must be supplied for registering bg worker', worker.name);
-        throw new Error('Name and run function must be supplied for registering bg worker ' + worker.name);
-    }
-    background_scheduler.run_background_worker(worker);
-}
+const register_bg_worker =
```

Comment:
I don't understand the point of this transformation.
You basically create a function to point to a function without doing any manipulation.
Why not just call the function directly?

Reply:
https://github.com/noobaa/noobaa-core/pull/6549#discussion_r649153223

Reply:
+ We want to avoid refractoring all the 21 places from `register_bg_worker` to `background_scheduler.register_bg_worker`